onnxruntime/python/tools/quantization/execution_providers/qnn/quant_config.py

```diff
@@ -0,0 +1,60 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from pathlib import Path
+
+import onnx
+
+from ...calibrate import CalibrationDataReader, CalibrationMethod
+from ...quant_utils import QuantType
+from ...quantize import StaticQuantConfig
+
+
+def get_qnn_qdq_config(
+    model_input: Path,
+    calibration_data_reader: CalibrationDataReader,
+    calibrate_method=CalibrationMethod.MinMax,
+    activation_type=QuantType.QUInt8,
+    weight_type=QuantType.QUInt8,
+    op_types_to_quantize=None,
+):
+    # Parse model nodes to setup overrides.
+    model = onnx.load_model(model_input)
+
+    tensor_quant_overrides = {}
+
+    for node in model.graph.node:
+        if node.op_type == "MatMul" and activation_type == QuantType.QUInt16:
+            tensor_quant_overrides[node.input[1]] = {"quant_type": QuantType.QUInt8}
```

Comment:
>QuantType.QUInt8 [](start = 67, length = 16)

should be TensorProto.UINT8
also need to add the import
from onnx import TensorProto

Reply:
I'm probably missing a call to QuantType.tensor_type somewhere in onnx_quantizer. I'm deliberately using QuantType so that the user can only set quant types exepcted by the quant script.